src/subscribables/dependentObservable.js

```diff
@@ -57,6 +57,34 @@ ko.computed = ko.dependentObservable = function (evaluatorFunctionOrOptions, eva
         isSleeping = false;
     }
 
+    function subscribeToDependency(target) {
+        if (target._deferUpdates) {
+            var dirtySub = target.subscribe(function () {
+                    // Process "dirty" events if both us and the target are using deferred updates
+                    if (dependentObservable._deferUpdates && !_isBeingEvaluated) {
+                        dependentObservable._evalRateLimited();
```

Comment:
Should _evalRateLimited be renamed, if it's now being used in non-rate-limted scenarios too?